### PR TITLE
Updated test by adding reference which might help with intermittent failures

### PR DESCRIPTION
**Pull request by jainaashish**
Updated test by adding reference which might help with intermittent failures of the test `InstallPackageAddImportStatement`

@rrelyea

### CODE REVIEW

File: test/EndToEnd/Packages/InstallPackageAddImportStatement/PackageWithImport.2.0.0.nuspec [hunk 1]
```diff
@@ -13,5 +13,6 @@
         <file src="build\PackageWithImport.targets" target="build\PackageWithImport.targets" />
         <file src="build\PackageWithImport.props" target="build\PackageWithImport.props" />
         <file src="content\one.txt" target="content\one.txt" />
+        <file src="lib\PackageWithImport.dll" target="lib\PackageWithImport.dll" />
```
Comment (rohit21agrawal) [inline test/EndToEnd/Packages/InstallPackageAddImportStatement/PackageWithImport.2.0.0.nuspec]: haha let's put it under a framework ? :P

**Review by rohit21agrawal — APPROVED** [review]

**Review by nkolev92 — APPROVED** [review]

**Comment by rohit21agrawal** [comment]
this test failed as part of your private build 8978. so still doesn't fix the issue

**Comment by jainaashish** [comment]
this is so weird, there must be something wrong with the Project System as it's not returning imports in some cases. May be we disable it for now.

**Comment by rohit21agrawal** [comment]
+1 to disabling it.

**Pull request merged**